locale/en.json

```diff
@@ -142,6 +142,26 @@
         "panel": "Your appeal application \n has been sent"
 
       }
+    },
+    "wrongHomeOfficeDecision": {
+      "title": "Home Office Decision",
+      "heading": "Why do you think the Home Office decision is wrong?",
+      "usefulInfo": "It may be helpful to look at the <a href='#'>Home Office documents</a> (opens in a new window). They include your decision letter and other information about your case.",
+      "textTitle": "<b>Tell us in your own words</b> - you don't have to have perfect English.",
+      "howToAnswer": "How to answer this question",
+      "bulletOne": "Carefully read your Home Office decision letter",
+      "bulletTwo": "Try to understand the parts of your claim the Home Office disagrees with",
+      "bulletThree": "Respond to some or all the things the Home Office disagrees with",
+      "bulletFour": "Tell us about anything <b>new</b> that will affect your case since you made your claim",
+      "bulletFive": "Include supporting evidence where you can. <b>You can do this on the next page</b>",
+      "needHelp": "If you need help",
+      "bulletLink": "Find organisations that can help you with your appeal"
+    },
+    "caseBuildingSuccess": {
+      "titleText": "Your answer has been sent",
+      "answer": "A case worker will look at your answer",
+      "date": "The case worker will contact you by <b>{{ date }}</b> to tell you what to do next",
+      "questions": "If you have any questions about the appeal process, call the Tribunal on <b>0300 123 1711</b> or email <a class=\"govuk-link govuk-link--no-visited-state\" href=\"mailto:customer.services@justice.gov.uk\">customer.services@justice.gov.uk</a>"
```

Comment:
copy on invision says 'If you have any question....'?

Reply:
Yh I checked with Frank and he said that was a typo... I'm guessing he will update soon.

Reply:
👍